### PR TITLE
fix: Backup points cannot be deleted when in use

### LINKED ISSUE #8: Backup points cannot be deleted when in use
**Describe the Bug**
When backup points are deleted, check that no backup files with backup points are used before they can be deleted; before performing backup operations, check whether the backup points exist, and disallow backups when an error is reported if they do not exist

**Versions Used**
Kubernetes:
Kubeclipper:

**Environment**
How many nodes and their hardware configuration:

For example: CentOS 7.5 / 3 masters:  8cpu/8g; 3 nodes: 8cpu/16g
(and other info are welcomed to help us debugging)

**How To Reproduce**

**Expected behavior**

### PR DESCRIPTION
**Pull request by Xvv-v**
Signed-off-by: Xvv-v <xu.jingwen@99cloud.net>

### What type of PR is this?
/kind bug

### Which issue(s) this PR fixes:

Fixes #8 

### Special notes for reviewers:
```
```

### Does this PR introduced a user-facing change?

```release-note

```

### Additional documentation, usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by x893675** [comment]
@zhuzhenfan @Metrora

File: pkg/apis/core/v1/utils.go [hunk 1]
```diff
@@ -363,3 +364,12 @@ func getActBackupStep(c *v1.Cluster, b *v1.Backup, bp *v1.BackupPoint, pNode *v1
 
 	return actBackup.GetStep(action), nil
 }
+
+func (h *handler) getBackupPoint(backups *v1.BackupList, name string) bool {
```
Comment (zhuzhenfan) [inline pkg/apis/core/v1/utils.go:367]: update func name “existBackupPoint”

Comment (Metrora) [reply pkg/apis/core/v1/utils.go]: I think maybe it would be better to change it to inUsed.

**Pull request merged**